fedmsg/tests/test_crypto_switching.py

```diff
@@ -40,10 +41,10 @@ def skip_if_missing_x509_libs(f):
     def _wrapper(self, *args, **kw):
         try:
             if six.PY3:
-                import cryptography
+                imp.find_module('cryptography')
```

Comment:
Same note as above about the ``imp`` module.

To get it to pass flake8 you can do ``import cryptography  # noqa: E402`` which I think is acceptable in this situation.